src/main/java/me/ironexception/universalfriends/configuration/Operations.java

```diff
@@ -12,19 +14,66 @@
+
+    /**
+     * Returns an immutable {@link Set} of all persons in the provided {@link Configuration} whose friendliness value fits the provided association.
+     *
+     *
+     * @param configuration The {@link Configuration} to mutate.
+     * @param association   The {@link Association} all {@link IPerson}s should have.
+     * @param <T>           The type of {@link IPerson} this configuration holds.
+     * @return              The mutated immutable {@link Configuration}.
+     */
+    public static <T extends IPerson> Set<T> getByAssociation(final Configuration<T> configuration, final Association association) {
+        if (association == Association.ALLY) return getFriends(configuration);
+        if (association == Association.ENEMY) return getEnemies(configuration);
+        return getByExactValue(configuration, association.getValue());
     }
 
-    public static <T extends IPerson> Set<T> getByAssociation(Configuration<T> configuration, Association association) {
-        double value = association.getValue();
-        return getByFriendliness(configuration, value);
+
+    /**
+     * Returns an immutable {@link Set} of all persons in the provided {@link Configuration} that are marked as friends.
+     * 
+     * @param configuration The {@link Configuration} to mutate.
+     * @param <T>           The type of {@link IPerson} this configuration holds.
+     * @return              The mutated immutable {@link Configuration}.
+     */
+    public static <T extends IPerson> Set<T> getFriends(final Configuration<T> configuration) {
+        return filterMatchingPersons(configuration, t -> t.getValue() > Standard.STANDARD_NEUTRAL);
+    }
+
+    /**
+     * Returns an immutable {@link Set} of all persons in the provided {@link Configuration} that are marked as enemies.
+     * 
+     * @param configuration The {@link Configuration} to mutate.
+     * @param <T>           The type of {@link IPerson} this configuration holds.
+     * @return              The mutated immutable {@link Configuration}.
+     */
+    public static <T extends IPerson> Set<T> getEnemies(final Configuration<T> configuration) {
+        return filterMatchingPersons(configuration, t -> t.getValue() < Standard.STANDARD_NEUTRAL);
     }
 
+    /**
+     * Returns an immutable {@link Set} of all persons in the provided {@link Configuration} that are marked as enemies.
+     * 
+     * @param configuration The {@link Configuration} to mutate.
+     * @param rangeLower    The minimum friendliness value that can occur in the resulting set. It is including.
+     * @param rangeUpper    The maximum friendliness value that can occur in the resulting set. It is including.
+     * @param <T>           The type of {@link IPerson} this configuration holds.
+     * @return              The mutated immutable {@link Configuration}.
+     */
     public static <T extends IPerson> Set<T> getInFriendlinessRange(Configuration<T> configuration, double rangeLower, double rangeUpper) {
```

Comment:
`getInFriendlinessRange` doesn't match this description. It returns friends in a certain range, and not just enemies.